src/screens/OperationDetails/Content.js

```diff
@@ -148,32 +225,7 @@ export default function Content({
           />
         </View>
 
-        {hasFailed || amount.isZero() ? null : (
-          <LText
-            semiBold
-            numberOfLines={1}
-            style={[styles.currencyUnitValue, { color: valueColor }]}
-          >
-            <CurrencyUnitValue
-              showCode
-              disableRounding={true}
-              unit={unit}
-              value={amount}
-              alwaysShowSign
-            />
-          </LText>
-        )}
-
-        {hasFailed || amount.isZero() ? null : (
-          <DoubleCounterValue
-            showCode
-            alwaysShowSign
-            currency={currency}
-            value={amount}
-            date={operation.date}
-            subMagnitude={1}
-          />
-        )}
+        {operationTitleRender()}
```

Comment:
a bit anti-pattern / overcomplex part

I would see this to be more a component than a function.

Reply:
Added a React.memo component for that 👌